src/components/FilterDrawer/TypeFilterGroup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  arrayOf, bool, func, shape, string,
+} from 'prop-types';
+import { StyleSheet, Text, View } from 'react-native';
+
+import TypeFilterRow from './TypeFilterRow';
+
+const TypeFilterGroup = ({
+  heading, filters, toggleResourceTypeFilterAction, disabled,
+}) => {
+  if (!filters.length) {
+    return null;
+  }
+  return (
+    <View>
+      <Text style={styles.heading}>{heading}</Text>
```

Comment:
Let's use `<BaseText />`. It has font size, color and weight prebuilt. It also uses the `variant` prop as a shortcut or you can provide your own `style={styles.whatever}` prop.

Reply:
I noticed that BaseText does not support an Array of values, and I was wondering about modifying it.

...this is how I'm using the `style` attribute, so that the styles, themselves, are more concise and clear.   (eg: you don't need to duplicate props from a "base" style into every style that overrides it)